### PR TITLE
Add Tokenlon space

### PR DESCRIPTION
**Pull request by BenjaminLu**
Tokenlon Website: https://tokenlon.im/instant
TFT Token Contract: https://etherscan.io/token/0xe534619defdbf0caf673b8abf7158714f5bd4bd9

**Add space for Tokenlon**  
  
Etherscan: https://etherscan.io/address/0xe534619defdbf0caf673b8abf7158714f5bd4bd9
CoinGecko: https://www.coingecko.com/en/exchanges/tokenlon
Website: https://tokenlon.im
Twitter: https://twitter.com/tokenlon

Github: https://github.com/consenlabs
Medium: https://medium.com/imtoken

### CODE REVIEW

File: skins/tokenlon.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.tokenlon {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (gokaiorg) [inline skins/tokenlon.scss:10]: If you use the default theme you will have to delete the scss file.

File: spaces/tokenlon/index.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "token": "0xE534619dEFDBF0cAf673b8AbF7158714F5BD4bd9",
+  "name": "Tokenlon",
+  "network": "1",
+  "symbol": "TFT",
+  "skin": "tokenlon",
```
Comment (gokaiorg) [inline spaces/tokenlon/index.json:6]: If you use the default theme you will have to remove the skin field.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/tokenlon/index.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "strategies": [
+    {
+      "name": "tokenlon",
```
Comment (bonustrack) [inline spaces/tokenlon/index.json:9]: This should be changed
```suggestion
      "name": "erc20-with-balance",
```

File: spaces/tokenlon/index.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "0x25A087dC3512e3BE548F0cA13233ae29d3781bdC",
+    "0x8Cccfa7B9EC9076b76041940D7835bC0A4488c6e",
+    "0x41964b3e8050B48DE5a1e8B0EFfe3c297BEB182E"
+  ],
+  "filters": {
+    "defaultTab": "all",
+    "minScore": 50,
+    "onlyMembers": false
+  },
+  "invalids": []
```
Comment (bonustrack) [inline spaces/tokenlon/index.json:27]: Invalids should be in "filters" but it's not necessary to add it if its empty.

**Comment by BenjaminLu** [comment]
Done. Much appreciated!

**Review by bonustrack — APPROVED** [review]

**Pull request merged**